corehq/apps/cloudcare/views.py

```diff
@@ -421,6 +423,23 @@ def get(self, request, *args, **kwargs):
         })
 
 
+def get_apps_by_user(domain, user_ids):
+    groups_by_user = get_groups_by_user(domain, user_ids)
+
+    app_access = ApplicationAccess.get_by_domain(domain)
```

Comment:
The app_access doc has the group_id and app_id, the user has its group memberships so it should be pretty easy to implement